README.md

```diff
@@ -5,52 +5,65 @@
+MLEM helps you package and deploy machine learning models.
+It saves model metadata in a standard, human-readable format that can be used in a variety of deployment scenarios, such as real-time serving through a REST API or a batch processing task.
+MLEM lets you keep Git as the single source of truth for code and models.
 
-- **Run your model anywhere you want:** package it as a Python package, a Docker Image or deploy it to Heroku (SageMaker, Kubernetes and more platforms are coming). Switch between formats and deployment platforms with a single command thanks to unified abstraction.
-- **Simple text file to save model metadata:** automatically package Python env requirements and input data specifications into a ready-to-deploy format. Use the same human-readable format for any ML framework.
-- **Stick to your training workflow:** MLEM doesn't ask you to rewrite your training code. To start using packaging or deployment machinery, add just two lines to your python script: one to import the library and one to save the model.
-- **Developer-first experience:** use CLI when you feel like DevOps and API when you feel like a developer.
+- **Run your ML models anywhere:**
+  Wrap models as a Python package or Docker Image, or deploy them to Heroku (SageMaker, Kubernetes, and more platforms coming soon).
+  Switch between platforms transparently, with a single command.
 
-## Why MLEM?
+- **Simple text file to save model metadata:**
+  Automatically include Python requirements and input data needs into a deployment-ready format.
+  Use the same format on any ML framework.
 
-The main reason to use MLEM instead of other related solutions is that it works well with **GitOps approach** and helps you manage model lifecycle in Git:
+- **Stick to your training workflow:**
+  MLEM doesn't ask you to rewrite model training code.
+  Just add two lines around your Python code: one to import the library and one to save the model.
 
-- **Git as a single source of truth:** we use plain text to save metadata for models that can be saved and versioned.
-- **Reuse existing Git and Github/Gitlab infrastructure** for model management instead of installing separate model management software.
-- **Unify model and software deployment.** Deploy models using the same processes and code you use to deploy software.
+- **Developer-first experience:**
+  Use the CLI when you feel like DevOps and the API when you feel like a developer.
+
+## Why is MLEM special?
+
+The main reason to use MLEM instead of other tools is to adopt a **GitOps approach**, helping you manage model lifecycles in Git:
+
+- **Git as a single source of truth:** MLEM writes model metadata to a plain text file that can be versioned in a Git repo along with your code.
```

Comment:
TBH I feel like this whole section could be merged in with the intro, as well as the bullet lists combined (not as-is).